### PR TITLE
Change: add missing nrt gui sprites

### PR DESCRIPTION
**Pull request by andythenorth**
The convert tools using the sprites from #9 aren't looking right.  

The existing rail convert icon is much higher fidelity, but I can't find it yet to figure why (oof).

### CODE REVIEW

**Comment by andythenorth** [comment]
* the first 2 icons in tram building toolbar are showing road
* the rail convert toolbar is included in sprites/png/gui/gui2x.png, but the equivalent road/tram convert icons are not provided at 2x yet, this looks inconsistent

**Comment by andythenorth** [comment]
Everything done as per list in #9, ready for review thx

File: sprites/extra/extra-openttd-gui.pnml [hunk 1]
```diff
@@ -319,3 +319,11 @@ replacenew ottd_gui175(OTTD_GUI, "", 175) {
 	[   44,   0,  20,  20,   0,   0, "sprites/png/gui/group_liveries.png"]         // boat group livery icon
 	[   66,   0,  20,  20,   0,   0, "sprites/png/gui/group_liveries.png"]         // aircraft group livery icon
 }
+replacenew ottd_gui179(OTTD_GUI, "", 179) { [   0,  0,  20,  20,   0,   0, "sprites/png/extra/extra-gui-build-tram.png"] } // global toolbar build tram
```
Comment (LordAro) [inline sprites/extra/extra-openttd-gui.pnml]: Could split these over a few more lines, like the above?

Comment (andythenorth) [reply sprites/extra/extra-openttd-gui.pnml]: Follows apparent house style per https://github.com/OpenTTD/OpenGFX/blob/aeeb2f3c17d79b2cb567c5da9d3654843f9f56d8/sprites/extra/extra-openttd-gui.pnml#L88

I don't like it, but eh

**Pull request merged**